quranHome/src/main/res/layout/juz_line.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:orientation="horizontal">
+
+    <View
+        android:id="@+id/startCircle"
+        android:layout_width="14dp"
+        android:layout_height="14dp"
+        android:layout_alignParentStart="true"
+        android:background="@drawable/ic_bg_juz_cirle_start" />
```

Comment:
These are vector drawables 
let's use `ImageView.srcCompat` to show them. this way we can benefit from appCompat functionality